### ISSUE TITLE
Wordpress shortcodes not working.

### ISSUE THREAD

**Issue by Constantech** [gantry/gantry5#2180]
I get the following errors in Chrome Console, which I do not get when using 2017 theme. The slider works with 2017 theme.
nextend-frontend.min.js?1510976657:1 Uncaught ReferenceError: n2 is not defined
at nextend-frontend.min.js?1510976657:1
(anonymous) @ nextend-frontend.min.js?1510976657:1
smartslider-frontend.min.js?1510976651:1 Uncaught ReferenceError: N2Require is not defined
at smartslider-frontend.min.js?1510976651:1
(anonymous) @ smartslider-frontend.min.js?1510976651:1
smartslider-simple-type-frontend.min.js?1510976653:1 Uncaught ReferenceError: N2Require is not defined
at smartslider-simple-type-frontend.min.js?1510976653:1
(anonymous) @ smartslider-simple-type-frontend.min.js?1510976653:1
n2-ss-9.js?1510984898:1 Uncaught ReferenceError: N2Require is not defined
at n2-ss-9.js?1510984898:1
Uncaught TypeError: N2Classes.SmartSliderSimple is not a constructor
at http://xxxx.com.au/xxxx/:147:530
at Object.<anonymous> (http://xxxx.com.au/xxxx/:141:376)
at i (http://xxxx.com.au/wp-includes/js/jquery/jquery.js?ver=1.12.4:2:27449)
at Object.fireWith [as resolveWith] (http://xxxx.com.au/wp-includes/js/jquery/jquery.js?ver=1.12.4:2:28213)
at Object.e.(anonymous function) [as resolve] (http://xxxx.com.au/wp-includes/js/jquery/jquery.js?ver=1.12.4:2:29192)
at HTMLDocument.<anonymous> (http://xxxx.com.au/wp-content/cache/nextend/web/n2/n2.js?1510977914:77:23)
at i (http://xxxx.com.au/wp-includes/js/jquery/jquery.js?ver=1.12.4:2:27449)
at Object.fireWith [as resolveWith] (http://xxxx.com.au/wp-includes/js/jquery/jquery.js?ver=1.12.4:2:28213)
at Function.ready (http://xxxx.com.au/wp-includes/js/jquery/jquery.js?ver=1.12.4:2:30006)
at HTMLDocument.K (http://xxxx.com.au/wp-includes/js/jquery/jquery.js?ver=1.12.4:2:30368)

**Comment by newkind** [comment]
Can you disable your plugins besides Gantry 5 and see if the site will start to work properly?

**Comment by Constantech** [comment]
Hi,

There's only 8 plugins installed, so I deactivated all but Gantry and the slider, and the slider worked.

I activated the other plugins one at a time, expecting to find the culprit in the process.  Got all the other plugins activated and the slider still worked with no errors in Chrome Inspector, so go figure on that one.  Also, there had been no updates to any plugin which might explain it.

Thanks very much for taking the time to consider this.

Kind Regards
John
